### DISCUSSION TITLE
Awards

### DISCUSSION THREAD

**Discussion by xlcrr** [OpenLitterMap/openlittermap-web#78] [Ideas]
As people map litter, we want them to unlock awards like

- First cigarette butt
- 100th cigarette butt
- User for 1 year 
- 1000 photos uploaded
- Litter mapped from each category 
- Litter mapped in 1,2,3+ countries
- You have achieved a position on the City / State / Country / Global leaderboard

What other kinds of awards would you like to integrate?

Developers - how do we make this work on the backend?

Do we have any graphic designer?

**Comment by FredMark** [comment]
"What other kinds of awards would you like to integrate?"
Some people commit in picking 1 piece of litter every day, so maybe something that would reward them, ('even' if they only pick 1 item):
- number of days in a row that you picked an item until today (set to zero when a day was skipped and start again (although it could be possible that he did pick litter, but uploads the day after, so be careful with this one)
- highest number of days in a row that you picked ever 
- number of days that you picked litter this year (set to zero at the beginning of a new year)

Maybe something with a 'litter-bingo' (looks like the "Litter mapped from each category", pick random objects (for instance things that you can find near a school) and award if you have find these

**Comment by xlcrr** [comment]
Today I was inspired on twitter- let’s name our awards after unsung heroes in society.

I struggle every day that everyone knows who celebrity A is, but few people know the name of Jonas Salk, who invented the polio vaccine and gave it away for free. [wikipedia](https://en.wikipedia.org/wiki/Jonas_Salk)

When you upload your first photo, you are sharing data just as Jonas Salk did. The Jonas Salk award ❤️

**Comment by FredMark** [comment]
Cool, nice way to pay a tribute to people who deserve it,
But then you probably also need an explanation somewhere who the person is/was?

**Reply by xlcrr** [reply]
Yes, the awards is going to be a huge part of the app.
